ocaml/xapi/xapi_vdi.ml

```diff
@@ -463,9 +463,8 @@ let snapshot ~__context ~vdi ~driver_params =
 			try
 				snapshot_and_clone C.VDI.snapshot ~__context ~vdi ~driver_params
 			with Storage_interface.Unimplemented _ ->
-				(* CA-28598 *)
 				debug "Backend reported not implemented despite it offering the feature; assuming this is an LVHD upgrade issue";
```

Comment:
Please delete the text "assuming this is an LVHD upgrade issue" from the debug log line, since this is no longer true, and could cause confusion to someone reading the log.